linux_os/guide/services/ssh/ssh_server/sshd_set_maxstartups/oval/shared.xml

```diff
@@ -0,0 +1,71 @@
+  </ind:textfilecontent54_object>
+  <ind:textfilecontent54_object id="obj_sshd_config_maxstartups_second_parameter" version="1">
+    <ind:filepath operation="equals">/etc/ssh/sshd_config</ind:filepath>
+    <ind:pattern operation="pattern match" datatype="string">(?i)^\s*MaxStartups\s+\d+:(\d+):\d+\s*$</ind:pattern>
+    <ind:instance operation="greater than or equal" datatype="int">1</ind:instance>
+  </ind:textfilecontent54_object>
+  <ind:textfilecontent54_object id="obj_sshd_config_maxstartups_third_parameter" version="1">
+    <ind:filepath operation="equals">/etc/ssh/sshd_config</ind:filepath>
+    <ind:pattern operation="pattern match" datatype="string">(?i)^\s*MaxStartups\s+\d+:\d+:(\d+)\s*$</ind:pattern>
+    <ind:instance operation="greater than or equal" datatype="int">1</ind:instance>
+  </ind:textfilecontent54_object>
+
+  <ind:textfilecontent54_state id="ste_sshd_config_start_parameter_valid" version="1">
+    <ind:subexpression datatype="int" operation="less than or equal">10</ind:subexpression>
+  </ind:textfilecontent54_state>
+  <ind:textfilecontent54_state id="ste_sshd_config_rate_parameter_valid" version="1">
+    <ind:subexpression datatype="int" operation="greater than or equal">30</ind:subexpression>
+  </ind:textfilecontent54_state>
+  <ind:textfilecontent54_state id="ste_sshd_config_full_parameter_valid" version="1">
+    <ind:subexpression datatype="int" operation="less than or equal">100</ind:subexpression>
+  </ind:textfilecontent54_state>
+
+  <ind:textfilecontent54_test check="all" check_existence="only_one_exists" id="tst_maxstartups_start_parameter" version="1" comment="SSH MaxStartups start parameter is less than or equal to 10">
+    <ind:object object_ref="obj_sshd_config_maxstartups_first_parameter" />
+    <ind:state state_ref="ste_sshd_config_start_parameter_valid" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_test check="all" check_existence="only_one_exists" id="tst_maxstartups_rate_parameter" version="1" comment="SSH MaxStartups rate parameter is greater than or equal to 30">
+    <ind:object object_ref="obj_sshd_config_maxstartups_second_parameter" />
+    <ind:state state_ref="ste_sshd_config_rate_parameter_valid" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_test check="all" check_existence="only_one_exists" id="tst_maxstartups_full_parameter" version="1" comment="SSH MaxStartups full parameter is less than or equal to 100">
+    <ind:object object_ref="obj_sshd_config_maxstartups_third_parameter" />
+    <ind:state state_ref="ste_sshd_config_full_parameter_valid" />
+  </ind:textfilecontent54_test>
+
+  <local_variable id="variable_sshd_config_maxstartups" datatype="int"
+  comment="Value of last MaxStartups from /etc/ssh/sshd_config" version="1">
+    <regex_capture pattern="^\s*MaxStartups\s+(\d+:\d+:\d+\s*$(?!\s*MaxStartups\s+(\d+:\d+:\d+\s*$)))">
+      <object_component item_field="subexpression"
+      object_ref="object_last_maxstartups_from_sshd_config" />
```

Comment:
We are not be able to merge this as is, this `object_ref` points to a non-existing object.

@alexhaydock Another alternative to merging this PR is for @cipherboy to push changes directly here, would you be okay with that?

Reply:
Hi @yuumasato - I've attempted to resolve the merge conflict here by removing the whole local variable. If this still isn't enough to let merging happen. If this still isn't enough then I'm happy for @cipherboy to push changes directly. Thanks again for your help :smile:

Reply:
Hi, sorry I was out yesterday.

The conflict with the CCEs file is solved. But some other rule was updated and took the same CCEs that were assigned to this rule. I could push a quick CCE update commit and merge this...

Reply:
> Hi, sorry I was out yesterday.
> 
> The conflict with the CCEs file is solved. But some other rule was updated and took the same CCEs that were assigned to this rule. I could push a quick CCE update commit and merge this...

Ah I'd missed that when resolving the conflict. Feel free to update the CCE list and merge if that is the easiest fix.

Reply:
@alexhaydock Sorry, taking a second look at the branch history and 8aa40ccb71a79876c97ba90427740c8728e3a1b7, there is something strange with the rebase. A lot of CEEs are showing as removed and that should not be the change set done by this PR.

Could you please try rebasing on top of latest master? And while you are at it, assign new CCEs to the rule?
(Sorry for the trouble, I haven't tried recently, but I think only original authors can do force-pushes to PRs)

Reply:
@yuumasato I'll take it over at some point. I just haven't had time yet. It will probably be next week after the holidays. I can clean it up then. :-)

Reply:
@yuumasato I think I fixed it. I've assigned new CCEs and managed to rebase on upstream properly. Not quite sure what went on with the resolution of that merge conflict earlier.

Reply:
I see what is strange, I think you are only updating the `cce-redhat-avail.txt`, the diff against master is ok, but there are commits removing a bunch of CEEs that were consumed in the master branch.

And now I see a merge from master into your branch, I'll try to clean the history.

Reply:
@yuumasato Would merging as a squash commit work if the diff against master is okay? Or have I managed to make too much of a mess of it? :facepalm:

Reply:
I got into a meeting right after I sent the message, :laughing: 

@alexhaydock I removed the merge commits and was able to force-push into your branch, check if it seems ok.

I just dropped the changes to `controls\cis_rhel7.yml`, 299380fdadd06d3d9c2688a23001bd688adf92e9 to avoid conflicts with #7134 and subsequent PRs @vojtapolasek has.

Reply:
@yuumasato No problem 😆 and thanks for sorting this out. I think just dropping this one from the in-progress RHEL7 control file is sensible for now. I know about the work @vojtapolasek has been doing there.

Reply:
@alexhaydock Cool, tests passed! I'm merging this one. :)